### PR TITLE
OEL-2756: Fix page teaser rendering.

### PR DESCRIPTION
**Pull request by AaronGilMartinez**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: themes/oe_showcase_theme/templates/node/node--oe-showcase-page--teaser.html.twig [hunk 1]
```diff
@@ -12,6 +12,6 @@
   title: _title,
   text: content.body,
   meta: [
-    node.getCreatedTime|bcl_timeago,
+    node.getCreatedTime|date("d F Y"),
```
Comment (brummbar) [inline themes/oe_showcase_theme/templates/node/node--oe-showcase-page--teaser.html.twig]: Please take a look at the other date formats.

**Review by brummbar — CHANGES_REQUESTED** [review]

File: themes/oe_showcase_theme/templates/node/node--oe-showcase-page--teaser.html.twig (hunk 1 above)
Comment (AaronGilMartinez) [reply themes/oe_showcase_theme/templates/node/node--oe-showcase-page--teaser.html.twig]: I applied same format than other dates but no specific date format for this one. Do you mean to create one?

Comment (drishu) [reply themes/oe_showcase_theme/templates/node/node--oe-showcase-page--teaser.html.twig]: yes, its best if we put these into config so they can be easly managed

File: modules/oe_showcase_search/tests/src/ExistingSite/IntegrationTest.php [hunk 2]
```diff
@@ -382,4 +402,18 @@ protected function assertElementsTexts(array $expected, array $elements): void {
     ));
   }
 
+  /**
+   * Asserts search result created date from items.
```
Comment (drishu) [inline modules/oe_showcase_search/tests/src/ExistingSite/IntegrationTest.php]: this description does not match what the function is doing

File: modules/oe_showcase_search/tests/src/ExistingSite/IntegrationTest.php [hunk 3]
```diff
@@ -213,6 +213,26 @@ public function testSearchFeature() {
 
     // Check search value is still present.
     $this->assertSame($search_input->getValue(), 'Imputo');
+
+    // Results display the created date.
+    $this->drupalGet('/search');
+    // First page are oe_showcase_page elements.
```
Comment (drishu) [inline modules/oe_showcase_search/tests/src/ExistingSite/IntegrationTest.php]: remove this comment, its redundant

File: modules/oe_showcase_search/tests/src/ExistingSite/IntegrationTest.php [hunk 4]
```diff
@@ -213,6 +213,26 @@ public function testSearchFeature() {
...
+    $this->assertResultsMeta([
+      '16 February 2022',
+      '14 January 2022',
+      '01 October 2021',
+      '01 October 2021',
+      '01 October 2021',
+    ]);
+    // Fourth page are oe_showcase_search_demo elements.
```
Comment (drishu) [inline modules/oe_showcase_search/tests/src/ExistingSite/IntegrationTest.php]: remove comment

**Review by drishu — CHANGES_REQUESTED** [review]

File: modules/oe_showcase_search/tests/src/ExistingSite/IntegrationTest.php (hunk 4 above)
Comment (AaronGilMartinez) [reply modules/oe_showcase_search/tests/src/ExistingSite/IntegrationTest.php]: Done

File: modules/oe_showcase_search/tests/src/ExistingSite/IntegrationTest.php (hunk 3 above)
Comment (AaronGilMartinez) [reply modules/oe_showcase_search/tests/src/ExistingSite/IntegrationTest.php]: Done

File: modules/oe_showcase_search/tests/src/ExistingSite/IntegrationTest.php (hunk 2 above)
Comment (AaronGilMartinez) [reply modules/oe_showcase_search/tests/src/ExistingSite/IntegrationTest.php]: Changed

File: themes/oe_showcase_theme/templates/node/node--search-result.html.twig [hunk 5]
```diff
@@ -20,6 +20,6 @@
   image: image,
   badges: _badges,
   meta: [
-    node.getCreatedTime|bcl_timeago,
+    node.getCreatedTime|date("d F Y"),
```
Comment (drishu) [inline themes/oe_showcase_theme/templates/node/node--search-result.html.twig]: you forgot to change this one

**Review by drishu — CHANGES_REQUESTED** [review]
also rebase pls

File: themes/oe_showcase_theme/templates/node/node--search-result.html.twig (hunk 5 above)
Comment (AaronGilMartinez) [reply themes/oe_showcase_theme/templates/node/node--search-result.html.twig]: Added

**Review by drishu — APPROVED** [review]

**Review by drishu — APPROVED** [review]

**Review by brummbar — APPROVED** [review]

**Pull request merged**